commands/deploy.go

```diff
@@ -1,20 +1,74 @@
 package commands
 
 import (
+	"bytes"
 	"fmt"
+	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
 // DeployCmd handles deploying an app to Section
-type DeployCmd struct{}
+type DeployCmd struct {
+	URL string
+}
 
 // Run deploys an app to Section's edge
 func (a *DeployCmd) Run() (err error) {
-	path, err := os.Getwd()
+	// path, err := os.Getwd()
+	// if err != nil {
+	// 	panic(err)
+	// }
+	path := "./"
+	var files []string
+	err = filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+		if strings.Contains(path, ".git/") {
+			return nil
+		}
+		if strings.Contains(path, ".lint/") {
+			return nil
+		}
+		files = append(files, path)
+		return nil
+	})
 	if err != nil {
-		panic(err)
+		return fmt.Errorf("failed to walk path: %v", err)
+	}
+	for _, file := range files {
+		fmt.Println(file)
 	}
+
 	fmt.Printf("Packaging %s\n", path)
+	var buf bytes.Buffer
```

Comment:
A temp file would be better otherwise this tool will consume as much ram as the size of their gzipped build output

Reply:
I can also use an `io.Pipe` and directly gzip to the wire and send the output to the server. Then it will not consume much ram and will not consume much disk space (if they are connected). I can change it to a temp file.

Reply:
Pipe is good too. Wasn't sure if you wanted to validate final tar before beginning upload (e.g. no file read errors, within size limit).

Reply:
Added max file size validation